### ISSUE TITLE
8.0.0b ApiTypeError: Invalid type for account subtype `ebt` and `other insurance` while getting accounts

### ISSUE THREAD

**Issue by imtibbet** [plaid/plaid-python#350]
`other insurance` is documented here
https://plaid.com/docs/api/accounts/#accounts-get-response-subtype_accounts

`ebt` is documented here
https://plaid.com/docs/api/accounts/#StandaloneAccountType-ebt_depository

error and another one just like it but with `(other insurance)` instead of `(ebt)`:
```
Invalid value for `value` (ebt), must be one of [None, '401a', '401k', '403B', '457b', '529', 'brokerage', 'cash isa', 'education savings account', 'gic', 'health reimbursement arrangement', 'hsa', 'isa', 'ira', 'lif', 'lira', 'lrif', 'lrsp', 'non-taxable brokerage account', 'other', 'prif', 'rdsp', 'resp', 'rlif', 'rrif', 'pension', 'profit sharing plan', 'retirement', 'roth', 'roth 401k', 'rrsp', 'sep ira', 'simple ira', 'sipp', 'stock plan', 'thrift savings plan', 'tfsa', 'trust', 'ugma', 'utma', 'variable annuity', 'credit card', 'paypal', 'cd', 'checking', 'savings', 'money market', 'prepaid', 'auto', 'business', 'commercial', 'construction', 'consumer', 'home', 'home equity', 'loan', 'mortgage', 'overdraft', 'line of credit', 'student', 'cash management', 'keogh', 'mutual fund', 'recurring', 'rewards', 'safe deposit', 'sarsep', 'null']
```

**Comment by phoenixy1** [comment]
fixed internally, fix will be deployed with next version release

**Comment by phoenixy1** [comment]
I believe this should be fixed in 8.0.0b(13) released today, please re-open if it is not.
